public/style.css

```diff
@@ -219,3 +219,169 @@ li:hover {
 }
 
 /*Welcome Page end*/
+/* RESULT PAGE UPTO -------- LINE-261 */
+.result {
+  width: 100vw;
+  height: 100vh;
+  background-image: url(https://wallpapercave.com/wp/wp8862729.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  color: white;
+  font-family: 'Designer Notes Bold', sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 7rem
+}
+
+.score {
+  object-fit: cover;
+  width: 200px;
+  height: 200px;
+  position: absolute;
+  top: 10px;
+  left: 40%;
+  transition: all 1s ease;
+  border-radius: 50%;
+  ;
+  animation: tilt 1s ease infinite;
+}
+
+.resultNum {
+  margin-top: 13rem;
+  display: flex;
+  align-items: center;
+  gap: 2rem;
+  width: 100%;
+  margin-left: 65%;
+}
+
+.complement {
+  font-size: 1.3rem;
+  color: rgb(0, 255, 47);
+  margin-top: 1.5rem;
+  font-family: 'Designer Notes', sans-serif;
+}
+
+.percentage {
+  width: 150px;
+  height: 150px;
+  top: 7rem;
+  left: 2rem;
+  border-radius: 50%;
+  background-image: url(https://us.123rf.com/450wm/sureshridle2/sureshridle21010/sureshridle2101000001/9175126-black-page-torn-from-center.jpg?ver=6);
+  background-size: cover;
+
+}
+
+.percentageNum {
+  margin-top: 4rem;
+  color: black;
+
+}
+
+.num-container {
+  display: flex;
+  height: 1rem;
+  gap: 2rem;
+  justify-content: center;
+  align-items: center;
+  background-color: white;
+  color: black;
+  padding: 10px;
+  border-radius: 10px;
+  transition: all 1s ease;
+}
+
+.num-container2 {
```

Comment:
Why is there a `num-container` and a `num-container2` class? Naming doesn't help me here

Reply:
just fixed it..